packages/apollo-server-plugin-response-cache/src/ApolloServerPluginResponseCache.ts

```diff
@@ -155,6 +155,10 @@ export default function plugin(
         'fqc:',
       );
 
+      const generateCacheKey = options.generateCacheKey
+        ? options.generateCacheKey
+        : (_: any, key: any) => sha(JSON.stringify(key));
```

Comment:
Why `any`? I think we can usually avoid `any`.

Note that you can probably delete the cacheKeyString function now.

Reply:
> Why `any`? I think we can usually avoid `any`.

fair point, I don't know why I left it that way. Updated it now with a type that makes it strongly typed.

> Note that you can probably delete the cacheKeyString function now.

it is gone now.

Reply:
thank your @trevor-scheer for the assist